packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksContent.tsx

```diff
@@ -26,6 +29,169 @@ const StyledEditable = styled(Editable)`
+    cursor: grabbing;
+    background: ${({ theme }) => theme.colors.neutral200};
+  }
+`;
+
+const NOT_DRAGGABLE_ITEMS = ['list'];
+const NOT_DROPPABLE_ITEMS = ['list'];
+
+type DragAndDropElementProps = {
+  children: RenderElementProps['children'];
+  index: Array<number>;
+  canDrag: boolean;
+  canDrop: boolean;
+};
+
+const DragAndDropElement = ({ children, index, canDrag, canDrop }: DragAndDropElementProps) => {
+  const { editor, disabled, name, setLiveText } = useBlocksEditorContext('drag-and-drop');
+  const { formatMessage } = useIntl();
+
+  const handleMoveBlock = React.useCallback(
+    (newIndex: Array<number>, currentIndex: Array<number>) => {
+      const [newNode] = Editor.node(editor, newIndex);
+      const [draggedNode] = Editor.node(editor, currentIndex);
+
+      Transforms.moveNodes(editor, {
+        at: currentIndex,
+        to: newIndex,
+      });
+
+      // Add 1 to the index for the live text message
+      const currentIndexPosition = [currentIndex[0] + 1, ...currentIndex.slice(1)];
+      const newIndexPosition = [newIndex[0] + 1, ...newIndex.slice(1)];
+
+      setLiveText(
+        formatMessage(
+          {
+            id: getTrad('components.Blocks.dnd.reorder'),
+            defaultMessage: '{item}, moved. New position in the editor: {position}.',
+          },
+          {
+            item: `${name}.${currentIndexPosition.join(',')}`,
+            position: `${newIndexPosition.join(',')} of ${editor.children.length}`,
+          }
+        )
+      );
+
+      // If a node is dragged out of the list block then convert it to a paragraph
+      if (newNode?.type !== 'list-item' && draggedNode?.type === 'list-item') {
+        Transforms.setNodes(editor, { type: 'paragraph' }, { at: newIndex });
+      }
+
+      // If a node is dragged into the list block then convert it to a list-item
+      if (newNode?.type === 'list-item' && draggedNode?.type !== 'list-item') {
+        if (newIndex[0] > currentIndex[0]) {
+          // Node is dragged downwards inside list
+          newIndex[0] -= 1;
+        }
+
+        const { type: _type, children: _children, ...extra } = draggedNode;
+        const attributesToClear: Record<string, null> = {};
+        Object.keys(extra).forEach((key) => {
+          attributesToClear[key] = null;
+        });
+
+        Transforms.setNodes(editor, { ...attributesToClear, type: 'list-item' }, { at: newIndex });
+      }
+    },
+    [editor, formatMessage, name, setLiveText]
+  );
+
+  const [{ handlerId, isDragging, handleKeyDown }, blockRef, dropRef, dragRef] = useDragAndDrop(
+    !disabled && canDrag,
+    {
+      type: `${ItemTypes.BLOCKS}._${name}`,
+      canDropHandler() {
+        return canDrop;
+      },
+      index,
+      item: {
+        displayedValue: children,
+      },
+      onMoveItem: handleMoveBlock,
+    }
+  );
+
+  const composedBoxRefs = composeRefs(blockRef, dropRef);
+
+  return (
+    <Box ref={composedBoxRefs}>
+      {isDragging ? (
```

Comment:
When dragging I can still see my cursor in the placeholder

https://github.com/strapi/strapi/assets/8087692/89c61219-4ef1-441e-863e-245554f4ec42

Reply:
fixed it